babel.config.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
+    plugins: [
+        '@babel/plugin-proposal-object-rest-spread',
+        [
+            'istanbul',
+            {
+                exclude: [
+                    '**/*.spec.js',
+                ],
+            },
+        ],
+    ],
+    presets: [
+        [
+            '@babel/preset-env',
+            {
+                modules: false,
+                exclude: [
+                    'transform-regenerator',
+                ],
+                targets: {
```

Comment:
Please find a way to target, include or use the data from our `.browserslistrc` from `dreiup-templates`

Reply:
this config setup is irrelevant since the components will be compiled with the `dreiup-templates/babel.config.js` anyways. This file exists just to compile the source files via webpack for the unit test.